app/views/article/record/_html_fulltext.erb

```diff
@@ -0,0 +1,13 @@
+<% doc_presenter = presenter(document) -%>
+<div class="section-container-heading" data-toggle="collapse" href="#toggleFulltext" aria-expanded="false" aria-controls="toggleFulltext">
+  <h2>Full text</h2>
+</div>
```

Comment:
@jvine and I talked about this a little bit yesterday. The `div` itself is not tab-able, but the second link (Hide / View full text) is.

The cursor does not indicate the `div` is clickable. Maybe I could turn it into a giant button?